### DISCUSSION TITLE
ivf container? output.ivf

### DISCUSSION THREAD

**Discussion by abitrolly** [xiph/rav1e#3128] [Q&A] [answered]
What is the output format listed in usage example? https://github.com/xiph/rav1e#compressing-video

Found https://wiki.multimedia.cx/index.php/IVF but that doesn't seem to be it.

**Comment by tdaede** [comment, marked as answer]
It's this container: https://wiki.multimedia.cx/index.php/Duck_IVF

It's an extremely simple demo container. Some games have shipped it but you should probably use MP4 or WebM instead, either by remuxing the ivf with an external tool or using rav1e via the API.

**Reply by abitrolly** [reply]
So it is video only as far as I can see. 

The description says it is for VP8 data. How does it pack AV1? Is using the Duck IVF for AV1 is widely supported by different tools?
